packages/css-engine/src/core/style-sheet.ts

```diff
@@ -119,6 +120,39 @@ export class StyleSheet {
     this.#cssText = css.join("\n");
     return this.#cssText;
   }
+  generateAtomic(options: { getKey: (rule: NestingRule) => string }) {
```

Comment:
so you killed the separation between atomic style sheet and regular style sheet and now there is only one style sheet that does both?

Reply:
I feel like this is a road towards a more dirty abstraction